test/integration_tests/configs/suites/quick/cls-er.json

```diff
@@ -5,6 +5,6 @@
   "dataset": "mnist.json",
   "backend": "local",
   "job_name": "class-incremental-mlp-cls-er",
-  "expected_accuracy_linux": [[0.9839243292808533, 0.9740450382232666]],
+  "expected_accuracy_linux": [[0.9834515452384949, 0.9740450382232666]],
```

Comment:
Shouldn't the results remain unaffected by the changes?

Reply:
See PR description for more details. In short, working with torch.tensor(1.0) and float(1.0) seem to cause differences.